### ISSUE TITLE
[BUG] Dropdown height issue on small screen devices in modal

### ISSUE THREAD

**Issue by NillerW** [kirbydesign/designsystem#3736]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
kirby-dropdown's content is not calculating height of content correctly, when displayed in a `kirby-modal` while applying the usePopover attribute on the `kirby-dropdown` component. This leads to the content of the dropdown potentially being cut-off for being off-screen, leading to last element(s) not being selectable.

## Describe how to reproduce the bug
1. Create a component that opens a `kirby-modal`
2. The component in the modal should implement `kirby-page-content` and inside it have a `kirby-dropdown`
3. Add `[userPopover]="true"` to `kirby-dropdown`
4. Simulate a small screen height and observe the dropdown's content being cut-off by the windows border.

## Which Kirby version was used?
10.1.3

## What was the expected behavior?
Content of dropdown adjusts its height to display all items within the windows display. 

## Add any screenshots
![image](https://github.com/user-attachments/assets/fe689e15-76b3-47bc-85a9-4b89800d1aa1)

## Please complete the following information:
- OS: MacOS
- Browser: Chrome
- Browser version: 130.0.6723.69

## Are there any additional context?
Reproduced on above mentioned specs. Also an issue on smaller tablets and phones when in landscape mode.

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
